README.md

```diff
@@ -322,6 +322,62 @@ Class['apt::update'] ->
 Class['::mysql::client']
 ```
 
+### Install MySQL 5.6 server on CentOS
+
+This minimal example shows how to install MySQL Community Server 5.6 on Centos 7.3 using Puppet 3.8.7 using hiera with puppetlabs-mysql=3.9.0
```

Comment:
Unless the version numbers are required (that is, the concept doesn't work with other version numbers), I would remove them from the basic description. Suggest changing to:

This example shows how to install MySQL Community Server on CentOS, using Puppet and Hiera.